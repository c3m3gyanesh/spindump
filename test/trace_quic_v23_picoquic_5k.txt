A test run with the picoquic client and server, for a 5K file transfer using the HTTP3 mode. Spinning only partially.
