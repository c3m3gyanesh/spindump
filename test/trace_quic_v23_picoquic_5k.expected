[
{ "Event": "new", "Type": "QUIC", "Addrs": ["172.16.95.160","34.238.92.221"], "Session": "08c399632b307146-cb4da3f9df52850b (62111:4433)", "Ts": 1571077363850720, "State": "Starting", "Packets1": 0, "Packets2": 0, "Bytes1": 0, "Bytes2": 0 },
{ "Event": "measurement", "Type": "QUIC", "Addrs": ["172.16.95.160","34.238.92.221"], "Session": "08c399632b307146-08b049ac4bcfaced (62111:4433)", "Ts": 1571077363930679, "State": "Up", "Right_rtt": 79959, "Packets1": 1, "Packets2": 0, "Bytes1": 1280, "Bytes2": 0, "Bandwidth1": 1280, "Bandwidth2": 0 },
{ "Event": "spinflip", "Type": "QUIC", "Addrs": ["172.16.95.160","34.238.92.221"], "Session": "c399632b307146a0-08b049ac4bcfaced (62111:4433)", "Ts": 1571077364007905, "State": "Up", "Transition": "0-1", "Who": "responder", "Packets1": 7, "Packets2": 8, "Bytes1": 3457, "Bytes2": 7808, "Bandwidth1": 3457, "Bandwidth2": 7808 }
]
